litebird_sim/distribute.py

```diff
@@ -0,0 +1,215 @@
+# -*- encoding: utf-8 -*-
+
+from collections import namedtuple
+
+Span = namedtuple("Span", ["start_idx", "num_of_elements"])
+"""A sub-range in a sequence of elements.
+
+It has two fields: `start_idx` and `num_of_elements`.
+
+"""
+
+
```

Comment:
I think that all the functions in this module can be avoided or simplified a lot by using `scipy.optimize.linear_sum_assignment`

Reply:
Mmm… I have never used that function, and it's difficult for me to understand how it works and what's the meaning of the return values. Can you suggest a patch that uses it? Ideally, you should put comments that explain these things, because I expect that other people might have the same problems as me in understanding it.

Reply:
Sure, I'll propose a single function for the two functions in this module

Reply:
I realized that function doesn't suit our case of use perfectly. Fine with your approach!